lib/appliance_console/internal_database_configuration.rb

```diff
@@ -84,6 +84,10 @@ def post_activation
       ServiceGroup.new(:internal_postgresql => true).restart_services
     end
 
+    def self.installed?
+      File.exist?(DATABASE_DISK_MOUNT_POINT.join("postgresql.conf"))
```

Comment:
@kbrock is this being used in here?  I don't know that a configuration file existing means PG is installed... What are your thoughts on where we'd use this?